flag.go

```diff
@@ -636,7 +636,7 @@ func withEnvHint(envVar, str string) string {
 			suffix = "%"
 			sep = "%, %"
 		}
-		envText = fmt.Sprintf(" [%s%s%s]", prefix, strings.Join(strings.Split(envVar, ","), sep), suffix)
```

Comment:
I find the original a bit easier to read here, but I'm guessing that you feel the opposite?